File: Java/String-2/src/countCode/CountCode.java
package countCode;

public class CountCode {
    //Retorne o número de vezes que a palavra "code" aparece na String dada, porém a letra "d" pode ser substituída por
    //qualquer uma, assim, valendo palavras como "core", "cofe" e "cowe".

    public int countCode(String str) {
        int cont = 0;
        for (int i = 0; i < str.length() - 3; i++) {
            if (str.startsWith("co",i) && str.substring(i+3,i+4).equals("e")) cont++;
        }
        return cont;
    }
}

File: Python/String-2/count_code.py
"""
Dada uma String "str", retorne o número de vezes em que a string "code" aparece,
contudo o "d" pode ser substituído por qualquer outra letra ("b", "z", "m"...).
Ex.:(('aaacodebbb') → 1; ('codexxcode') → 2; ('cozexxcope') → 2).
"""


def count_code(str):
    cont = 0
    for i in range(len(str)):
        if str[i:i+2] == "co" and str[i+3:i+4] == "e":
            cont += 1
    return cont


print(count_code("cofe"))
